cabal.project

```diff
@@ -187,9 +187,7 @@ constraints:
   , io-classes >= 1.4
   , io-classes -asserts
 
-  , ouroboros-consensus-cardano ^>= 0.18
-  , ouroboros-network ^>= 0.16.1
-  , ouroboros-network-protocols ^>= 0.9
+  , ouroboros-network ^>= 0.17
```

Comment:
I would feel more comfortable with more heavy pinning, for example also

```
                      , ouroboros-consensus-cardano ^>= 0.19
```

— this way, we statically rule out that we draw in an old dependency. But I'm also ok with this.